app/Makefile

```diff
@@ -1,15 +1,21 @@
+all:
+	@echo "run:			meteor npm start"
+	@echo "parse-server:		meteor npm run server"
+	@echo "run-test:		meteor npm run test"
+	@echo "test-watch:		meteor npm run test-watch"
+
 # Dev server
 run:
-	npm run start
+	meteor npm start
```

Comment:
Thanks @jgoley - if you can get the conflicts resolved I will go ahead and merge it